### PR TITLE
RSA15c

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by mattheworiordan** [comment]
LGTM

**Comment by tcard** [comment]
PTAL

File: Spec/Auth.swift [hunk 1]
```diff
@@ -693,17 +692,17 @@ class Auth : QuickSpec {
 
                 it("overriding defaults") {
                     // Custom values
-                    let expectedClientId = "token_client"
                     let expectedTtl = 4800.0
                     let expectedCapability = "{\"canpublish:*\":[\"publish\"]}"
 
-                    let tokenParams = ARTTokenParams(clientId: expectedClientId)
+                    let tokenParams = ARTTokenParams(clientId: currentCliend)
```
Comment (ricardopereira) [inline Spec/Auth.swift]: I don't understand the `currentCliend`. Where does it come from?

**Comment by ricardopereira** [comment]
Beside a small comment, LGTM.

**Comment by ricardopereira** [comment]
2ff8945 will close #126. 👍

File: Spec/Auth.swift (hunk 1 above)
Comment (tcard) [inline Spec/Auth.swift]: It's the client ID passed to the `rest` instance used below. RSA15c says that the client ID can't change in a TokenRequest or TokenDetails from the one passed in the ClientOptions, so we can't expect it to be different.

Comment (ricardopereira) [inline Spec/Auth.swift]: Oh, I see now. Sorry. Can you change it to `currentClient`?

Comment (tcard) [inline Spec/Auth.swift]: Done at 52b87bb.

Comment (ricardopereira) [inline Spec/Auth.swift]: Thanks.

**Pull request merged**